packages/orbit-components/src/ButtonMobileStore/index.tsx

```diff
@@ -41,16 +30,17 @@ const ButtonMobileStore = ({
   };
 
   return (
-    <StyledButtonMobileStore
+    <a
+      className="inline-block h-[40px]"
       href={href}
       target="_blank"
-      rel="noopener"
+      rel="noopener noreferrer"
```

Comment:
That was not added for reason, [read more here ](https://skypicker.slack.com/archives/CAMS40F7B/p1639150535221300) ;)